### PR TITLE
Translate helm/version_skew(Helm Version Support Policy) in Korean

### PR DESCRIPTION
**Pull request by bluefriday**
Signed-off-by: bluefriday <bluefriday86@gmail.com>
Translate helm/version_skew(Helm Version Support Policy) in Korean
- https://helm.sh/docs/topics/version_skew/

### CODE REVIEW

**Comment by bluefriday** [comment]
@sunminjeon @jmnote
리뷰 부탁드립니다.

File: content/ko/docs/helm/version_skew.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+헬름은 상위 호환성을 보장하지 않으므로, 컴파일 
+된 버전보다 새로운, 쿠버네티스 버전과 함께 
+헬름을 사용하지 않는 것이 좋다.
+
+만약 지원하지 않는 쿠버네티스 버전과 함께 헬름을 사용하기로 선택한 경우, 
+사용자는 위험을 감수하고 사용하게 되는 것이다.
+
+클러스터와 호환되는 헬름 버전을 확인하려면 아래의 표를 
+참조한다.
```
Comment (jmnote) [inline content/ko/docs/helm/version_skew.md:45]:
```suggestion
클러스터와 호환되는 헬름 버전을 확인하려면 아래의 표를 
참조하자.
```

File: content/ko/docs/helm/version_skew.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+헬름 프로젝트는 최신 부 릴리스에 대한 릴리스 분기를 유지한다.
+보안 수정을 포함한 적용 가능한 수정은 심각도와 실행 가능성에 따라
+릴리스 브랜치에 선별된다. 패치 릴리스는 필요에 따라 해당 분기에서 
+잘라낸다. 이 결정은 릴리스 관리자가 소유한다.
+
+## 지원되는 버전 차이(skew)
+
+헬름의 새 버전이 출시 되면, 쿠버네티스의 특정 부 버전에 
+대해 컴파일 된다. 예를 들어 헬름 3.0.0은 쿠버네티스 1.16.2 클라이언트를 사용하여 
+쿠버네티스와 상호 작용하므로 쿠버네티스 1.16 과 호환된다.
```
Comment (jmnote) [inline content/ko/docs/helm/version_skew.md:24]:
```suggestion
헬름의 새 버전이 출시 되면, 쿠버네티스의 특정 부 버전에 
대해 컴파일된다. 예를 들어 헬름 3.0.0은 쿠버네티스 1.16.2 클라이언트를 사용하여 
쿠버네티스와 상호작용하므로 쿠버네티스 1.16 과 호환된다고 할 수 있다.
```

File: content/ko/docs/helm/version_skew.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+헬름 3부터, 헬름은 컴파일 된 쿠버네티스의 `n-3` 버전과 호환되는 
+것으로 간주된다. 마이너 버전 간의 쿠버네티스 변경으로 인해, 헬름 2의 
+지원 정책은 쿠버네티스의 `n-1` 버전과 호환된다고 가정하여
+약간 더 엄격하다.
```
Comment (jmnote) [inline content/ko/docs/helm/version_skew.md:29]:
```suggestion
헬름 3부터는, 헬름은 그에 대응하여 컴파일되는 쿠버네티스 `n-3` 버전과 호환되는 
것으로 간주된다. 쿠버네티스 마이너 버전 간의 차이로 인해 헬름 2의 
지원 정책은 약간 더 엄격한데, 쿠버네티스의 `n-1` 버전과 호환되는 것으로
간주한다.
```

File: content/ko/docs/helm/version_skew.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+및 1.14 와 함께 사용하는 것이 안전하다. 쿠버네티스 1.16 클라이언트 
+API 에 대해 컴파일 된 헬름 2 버전을 사용하는 경우 
+쿠버네티스 1.16 및 1.15 와 함께 사용하는 것이 안전하다.
+
+헬름은 상위 호환성을 보장하지 않으므로, 컴파일 
+된 버전보다 새로운, 쿠버네티스 버전과 함께 
+헬름을 사용하지 않는 것이 좋다.
+
+만약 지원하지 않는 쿠버네티스 버전과 함께 헬름을 사용하기로 선택한 경우, 
+사용자는 위험을 감수하고 사용하게 되는 것이다.
```
Comment (jmnote) [inline content/ko/docs/helm/version_skew.md:42]:
```suggestion
만약 지원하지 않는 쿠버네티스 버전에서 헬름을 사용하고자 하는 경우, 
사용자는 위험을 감수하고 사용하게 된다.
```

File: content/ko/docs/helm/version_skew.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+예를 들어 쿠버네티스 1.17 클라이언트 API 에 대해 컴파일 
+된 헬름 3 버전을 사용하는 경우 쿠버네티스 1.17, 1.16, 1.15 
+및 1.14 와 함께 사용하는 것이 안전하다. 쿠버네티스 1.16 클라이언트 
+API 에 대해 컴파일 된 헬름 2 버전을 사용하는 경우 
+쿠버네티스 1.16 및 1.15 와 함께 사용하는 것이 안전하다.
+
+헬름은 상위 호환성을 보장하지 않으므로, 컴파일 
+된 버전보다 새로운, 쿠버네티스 버전과 함께 
+헬름을 사용하지 않는 것이 좋다.
```
Comment (jmnote) [inline content/ko/docs/helm/version_skew.md:39]:
```suggestion
헬름은 상위호환성을 보장하지 않으므로, 대응하여 컴파일된
버전보다 더 높은 쿠버네티스 버전에서
헬름을 사용하는 것은 권장되지 않는다.
```

File: content/ko/docs/helm/version_skew.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+헬름 3부터, 헬름은 컴파일 된 쿠버네티스의 `n-3` 버전과 호환되는 
+것으로 간주된다. 마이너 버전 간의 쿠버네티스 변경으로 인해, 헬름 2의 
+지원 정책은 쿠버네티스의 `n-1` 버전과 호환된다고 가정하여
+약간 더 엄격하다.
+
+예를 들어 쿠버네티스 1.17 클라이언트 API 에 대해 컴파일 
+된 헬름 3 버전을 사용하는 경우 쿠버네티스 1.17, 1.16, 1.15 
+및 1.14 와 함께 사용하는 것이 안전하다. 쿠버네티스 1.16 클라이언트 
+API 에 대해 컴파일 된 헬름 2 버전을 사용하는 경우 
+쿠버네티스 1.16 및 1.15 와 함께 사용하는 것이 안전하다.
```
Comment (jmnote) [inline content/ko/docs/helm/version_skew.md:35]:
```suggestion
예를 들어, 쿠버네티스 1.17 클라이언트 API 에 대응하여 컴파일된
헬름 3 버전을 사용하는 경우, 쿠버네티스 1.17, 1.16, 1.15 
및 1.14 와 함께 사용하는 것이 안전할 것이다. 쿠버네티스 1.16 클라이언트 
API 에 대해 컴파일 된 헬름 2 버전을 사용하는 경우, 
쿠버네티스 1.16 및 1.15 와 함께 사용하는 것이 안전할 것이다.
```

File: content/ko/docs/helm/version_skew.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+된 버전보다 새로운, 쿠버네티스 버전과 함께 
+헬름을 사용하지 않는 것이 좋다.
+
+만약 지원하지 않는 쿠버네티스 버전과 함께 헬름을 사용하기로 선택한 경우, 
+사용자는 위험을 감수하고 사용하게 되는 것이다.
+
+클러스터와 호환되는 헬름 버전을 확인하려면 아래의 표를 
+참조한다.
+
+| 헬름 버전       | 지원되는 쿠버네티스 버전              |
```
Comment (jmnote) [inline content/ko/docs/helm/version_skew.md:47]:
```suggestion
| 헬름 버전       | 지원하는 쿠버네티스 버전              |
```

File: content/ko/docs/helm/version_skew.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
+---
+title: "헬름 버전 지원 정책"
+description: "헬름의 패치 릴리스 정책과 헬름, 헬름과 쿠버네티스 간에 지원되는 최대 버전 차이를 설명"
+---
+
+이 문서는 헬름과 쿠버네티스 간에 지원되는 최대 버전의 차이를
+설명한다.
```
Comment (jmnote) [inline content/ko/docs/helm/version_skew.md:7]:
```suggestion
이 문서는 헬름과 쿠버네티스 간에 지원되는 최대의 버전 차이를
설명한다.
```

File: content/ko/docs/helm/version_skew.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
+---
+title: "헬름 버전 지원 정책"
+description: "헬름의 패치 릴리스 정책과 헬름, 헬름과 쿠버네티스 간에 지원되는 최대 버전 차이를 설명"
+---
```
Comment (jmnote) [inline content/ko/docs/helm/version_skew.md:4]:
```suggestion
---
title: "헬름 버전 지원 정책"
description: "헬름의 패치 릴리스 정책과 헬름, 헬름과 쿠버네티스 간에 지원되는 최대의 버전 차이를 설명"
---
```

**Review by jmnote — COMMENTED** [review]
수고하셨습니다.

**Comment by bluefriday** [comment]
> 수고하셨습니다.

@jmnote 감사합니다. 의견 주신 부분 반영하였습니다.

**Comment by jmnote** [comment]
lgtm

**Comment by bridgetkromhout** [comment]
The file looks good, but the path is incorrect. The `docs/helm` path is for the autogenerated docs. This file should be in `docs/topics` similarly to this path: https://github.com/helm/helm-www/blob/master/content/en/docs/topics/version_skew.md, putting it into this directory: https://github.com/helm/helm-www/tree/master/content/ko/docs/topics

**Review by singlemocha — COMMENTED** [review]
LGTM

**Comment by bluefriday** [comment]
> The file looks good, but the path is incorrect. The `docs/helm` path is for the autogenerated docs. This file should be in `docs/topics` similarly to this path: https://github.com/helm/helm-www/blob/master/content/en/docs/topics/version_skew.md, putting it into this directory: https://github.com/helm/helm-www/tree/master/content/ko/docs/topics

@bridgetkromhout Thank you for your detailed review. I changed to corrected file path :D

**Review by bridgetkromhout — APPROVED** [review]

**Pull request merged**